src/main/scala/scalafix/sbt/ScalafixEnable.scala

```diff
@@ -57,7 +57,7 @@ object ScalafixEnable {
       )
     } yield setting
 
-    val semanticdbInstalled = extracted.append(settings, s)
+    val semanticdbInstalled = extracted.appendWithoutSession(settings, s)
```

Comment:
ahh didn't noticed that this was still being tested against sbt 0.13.x @olafurpg do you want to keep it?

Reply:
Yes, we should keep this for compatibility with 0.13.